src/frontend/src/pages/ChangeRequestDetailPage/ImplementedChangesList.tsx

```diff
@@ -6,15 +6,58 @@
 import { ImplementedChange } from 'shared';
 import { datePipe, emDashPipe, fullNamePipe, wbsPipe } from '../../utils/pipes';
 import { routes } from '../../utils/routes';
-import { Link, ListItem, List, Tooltip, Typography } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { Link, ListItem, List, Tooltip, Typography, TooltipProps, tooltipClasses } from '@mui/material';
 import PageBlock from '../../layouts/PageBlock';
+import { useLayoutEffect, useState } from 'react';
 
 interface ImplementedChangesListProps {
   changes: ImplementedChange[];
   overallDateImplemented?: Date;
 }
 
 const ImplementedChangesList: React.FC<ImplementedChangesListProps> = ({ changes, overallDateImplemented }) => {
+  const [bodyWidth, setBodyWidth] = useState<number>(window.document.body.offsetWidth);
+
+  window.document.body.addEventListener('resize', () => {
+    setBodyWidth(window.document.body.offsetWidth);
+  });
+
+  function useWindowSize() {
+    const [innerWidth, setInnerWidth] = useState<number>(0);
+    function determinePosition() {
+      return window.innerWidth <= bodyWidth ? 'top' : 'right';
```

Comment:
Looks like you're comparing current window width to window width upon loading. So for me that means tooltips by default appear on top (`(window.innerwidth === bodyWidth) -> 'top'`) unless I shrink the page with Ctrl/Cmd+minus. We want the tooltip to display `right` always unless the tooltop goes beyond the width of the screen.

Reply:
Sorry, just getting around to this. Seems like it's no longer working for me either so maybe it working a few weeks ago was a fluke. Since there is also word-wrapping right now, not sure if it would be worth pushing the word wrapping + always keeping it at the top to prevent overflow, and then trying to debug switching the positioning.